blocks_common/math.js

```diff
@@ -22,15 +22,8 @@
  * @fileoverview Math blocks for Blockly.
  * @author q.neutron@gmail.com (Quynh Neutron)
  */
-'use strict';
-
-goog.provide('Blockly.Blocks.math');
-
-goog.require('Blockly.Blocks');
-
-goog.require('Blockly.Colours');
-
-goog.require('Blockly.constants');
+import * as Blockly from 'blockly';
```

Comment:
Here and below.

```suggestion
import * as Blockly from 'blockly/core';
```

The `blockly` package used to be a "batteries included" version of Blockly that came with some messages and code generators, but it has degraded over time. The [current recommended approach](https://developers.google.com/blockly/guides/get-started/get-the-code) is to import from core if you're just looking for the guts of Blockly, and then import the other things (messages, etc) as necessary.